pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -142,6 +142,22 @@ let
         };
       };
 
+      adzero.vscode-sievehighlight = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "vscode-sievehighlight";
+          publisher = "adzero";
+          version = "1.0.6";
+          hash = "sha256-8Ompv792eI2kIH+5+KPL9jAf88xsMGQewHEQwi8BhoQ=";
+        };
+        meta = {
+          changelog = "https://marketplace.visualstudio.com/items/adzero.vscode-sievehighlight/changelog";
+          description = " Syntax highlight support for Sieve mail filtering language.";
```

```suggestion
          description = "Syntax highlight support for Sieve mail filtering language";
```
- No `.` at end of the description.
- `A Visual Studio Code extension` is missing.
- There was a space at beginning of description.

---

Optional: Consider adding yourself as maintainer to be notified on updates.